src/pages/index.ts

```diff
@@ -1,7 +1,8 @@
+import { SinglePostPage } from './SinglePostPage';
 import { BookMarks } from './BookMarks';
 import { Profile } from './Profile';
 import { Home } from 'pages/Home';
 import { SignUp } from './SignUp';
 import { Login } from 'pages/Login';
 import { Explore } from 'pages/Explore';
-export {Explore,Login,SignUp,Home,Profile,BookMarks}
\ No newline at end of file
+export {Explore,Login,SignUp,Home,Profile,BookMarks,SinglePostPage}
```

Comment:
you can use export { explore } from '....' directly here, instead of first importing and then exporting

Reply:
Thanks for pointing it will do from next time